docs/product/others/websocket-events.md

```diff
@@ -470,3 +470,142 @@ This event would be available for both agent and the contact, it returns the ava
   }
 }
 ```
+
+### notification_created
+
+> Available for: Agents
+
+```json
```

Comment:
@muhsin-k There is an update in the document. Please add a notification object at the top, and update the rest of the document similar to other events in the document.